.travis.yml

```diff
@@ -12,4 +12,8 @@ jobs:
       before_install: docker pull thesofproject/sof && docker tag thesofproject/sof sof
       script:
         - echo -e '#!/bin/bash\nmkdir build && cd build && cmake .. && make' > build.sh && chmod +x build.sh
-        - docker run -i -t -v `pwd`:/home/sof/work/sof.git --user `id -u` sof ./build.sh
\ No newline at end of file
+        - docker run -i -t -v `pwd`:/home/sof/work/sof.git --user `id -u` sof ./build.sh
+    - name: checkpatch
+      script:
+        - printf '%s, %s\n' "${TRAVIS_BRANCH}" "${TRAVIS_COMMIT_RANGE/.../..}"
+        - (set x; scripts/checkpatch.pl --no-tree --strict --codespell --no-signoff -g ${TRAVIS_COMMIT_RANGE/.../..})
```

Comment:
I guess you copied the `set -x` trick from the one I added in `linux/.travis.yml`. The purpose of this `set -x` trick is to show the expanded value of `${TRAVIS_COMMIT_RANGE/.../..}`, which makes the `printf` unnecessary.

If you think showing the value of `"${TRAVIS_BRANCH}"` is useful too, then you might as well make it really useful and run a short `git log` instead, see example at: https://travis-ci.org/github/thesofproject/linux/jobs/728647259

Reply:
Updated